validation/glean/baseline.1.pass.json

```diff
@@ -1,98 +0,0 @@
-{
```

Comment:
Don't we want to just rename (not remove) this file?

Reply:
I wasn't sure if it exercised anything different than the other test file... if it does I can rename it instead

Reply:
the .all appear to be inclusive so +1 for the removal

Reply:
Ah, actually that's a good catch.  I think they've become more-or-less redundant at this point, and `baseline.1.all.pass.json` is the more complete and better to keep one.

Reply:
I think we should keep `baseline.1.pass.json` and remove `baseline.1.all.pass.json`. The former is a ping we are actually likely to see, and the second is not. Now that we have: baseline, events, and metrics samples, the `all` seems redundant.

Reply:
`baseline.1.all.pass.json` exercises all of the metric types, but `metrics.1.all.pass.json` (despite the name) doesn't.  Maybe we keep `baseline.1.pass.json` (since it's a realistic example) and remove `metrics.1.all.pass.json` and rename `baseline.1.all.pass.json` to `metrics.1.all.pass.json`.  (Sorry this all got so messy ... :)

Reply:
@mdboom I agree with that assessment. Best to match what we're actually sending

Reply:
Ok, I think I've got the above. I kept the `"experiments"` section from `metrics.1.all.pass.json`, but otherwise replaced its contents with `baseline.1.pass.json`. I also removed the validation examples in `org-mozilla-samples-glean/` since they were repeats of the stuff in the `glean/` dir, which are straight-up template applications of the main `glean` schema.

@mdboom does this look ok now?

Reply:
Yep.  Thanks.  :+1: